src/main/java/org/kohsuke/github/GHEventPayload.java

```diff
@@ -1635,68 +1235,11 @@ void wrapUp(GitHub root) {
      *      repository event</a>
      * @see <a href="https://docs.github.com/en/rest/reference/repos">Repositories</a>
      */
-    @SuppressFBWarnings(
-            value = { "UWF_FIELD_NOT_INITIALIZED_IN_CONSTRUCTOR", "NP_UNWRITTEN_FIELD", "UWF_UNWRITTEN_FIELD" },
-            justification = "Constructed by JSON deserialization")
     public static class Repository extends GHEventPayload {
-        private String action;
-        private GHRepository repository;
-        private GHOrganization organization;
-
-        /**
-         * Gets action.
-         *
-         * @return the action
-         */
-        public String getAction() {
-            return action;
-        }
-
-        /**
-         * Sets repository.
-         *
-         * @param repository
-         *            the repository
-         */
-        public void setRepository(GHRepository repository) {
-            this.repository = repository;
-        }
-
-        /**
-         * Gets repository.
-         *
-         * @return the repository
-         */
-        public GHRepository getRepository() {
-            return repository;
-        }
-
-        /**
-         * Gets organization.
-         *
-         * @return the organization
-         */
-        public GHOrganization getOrganization() {
-            return organization;
-        }
-
-        /**
-         * Sets organization.
-         *
-         * @param organization
-         *            the organization
-         */
-        public void setOrganization(GHOrganization organization) {
-            this.organization = organization;
-        }
-
         @Override
         void wrapUp(GitHub root) {
             super.wrapUp(root);
             repository.wrap(root);
```

Comment:
The `super` handles the `repository.wrap()`, right?

```suggestion
        void wrapUp(GitHub root) {
            super.wrapUp(root);
```

Reply:
done. the inherited method removed (contains only a call to super)